src/scenes/home/informationForm/informationForm.js

```diff
@@ -16,6 +16,7 @@ class SignupInformation extends Component {
     super(props);
     this.onIdentifierStatusChange = this.onIdentifierStatusChange.bind(this);
     this.onCheckBoxChange = this.onCheckBoxChange.bind(this);
+    this.previousPage = this.previousPage.bind(this);
```

Comment:
Yes, just like we bind the 'onCheckboxChange' function so we can reference the state through 'this'